autograder/utils/testing/model_obj_builders.py

```diff
@@ -225,6 +225,14 @@ def make_finished_submission(group: Optional[ag_models.Group] = None,
                            **submission_kwargs)
 
 
+def make_rejected_submission(group: Optional[ag_models.Group] = None,
```

Comment:
Is this still being used, or can we remove it?

Reply:
We can delete this.